deploy/bz2067978/01-osd-delete-noderesolver-daemonset.Role.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: osd-delete-noderesolver-daemonset-bz2067978
```

Comment:
This role is named the same as the the other one.. should this be combined?

Reply:
The distinguishing factor is that one role sits in `openshift-dns` and the other in `openshift-dns-operator` (as it has to do things in both)

Reply:
Oh I missed that. Thanks!